extensions/EXT/EGL_EXT_protected_content.txt

```diff
@@ -200,6 +211,11 @@ Additions to Chapter 3 of the EGL 1.4 Specification (Rendering Surfaces)
 
         The default value of EGL_PROTECTED_CONTENT_EXT is EGL_FALSE."
 
+    Add the following paragraph in section 3.7.4 Context Queries. Add after
+    the last paragraph after eglQueryContext queries.
+    
+        "Querying EGL_PROTECTED_CONTENT_EXT returns the current value"
+
```

Comment:
These additions look like they are the wrong way round in the PR to me. That is, the addition to section 3.5.6 follows the part about adding to section 3.7 and this one, which adds to section 3.7.4 follows the part about adding to section 3.5. (I don't know why the original paragraphs were in reverse order).

Reply:
OK. I will check that. thx.

Reply:
Thanks. Apart from that, I've reviewed this and I'm fine with it.

Reply:
I sure did get them mixed. In a rush I guess. thanks - fixed.